sdk/storage/azure-storage-queue/azure/storage/queue/_shared/constants.py

```diff
@@ -0,0 +1,24 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import sys
+
+
+X_MS_VERSION = '2018-03-28'
```

Comment:
This seems to duplicate  the version in https://github.com/Azure/azure-sdk-for-python/pull/6039/files#diff-8b893eaf830a6ba5ad078c41e67ba67b could there be a single source of truth for these? (Or are they only coincidentally identical at the moment)

Reply:
Agreed - I have opened an issue here:
https://github.com/Azure/azure-sdk-for-python/issues/6193